src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  &__container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+
+    width: 4 * 200px + 3 * 48px;
```

Comment:
Let's move the number of cards to the specific variable, so we could change it easily later on.

Reply:
Thx!
Done